roles/frontend-test/tasks/main.yaml

```diff
@@ -0,0 +1,4 @@
+- name: Run frontend unittests
```

Comment:
add meta/main.yaml
with 
dependencies:
  - role: ensure-npm

and implement the role ensure-npm that install npm